content/topics/web-frontend/react/redux-architecture/_index.md

```diff
@@ -1,5 +1,18 @@
 ---
 _db_id: 515
 content_type: topic
+prerequisites:
+  hard:
+  - topics/redux-intro/
+  soft: []
+ready: true
 title: React + Redux architectural guidelines
----
\ No newline at end of file
+---
+
+Redux is a very powerful state management library used in JavaScript apps to mange the data of your application and was originally designed for react. Redux helps you write applications that behave consistently and can run in different environments. 
```

Comment:
Looks good, just fix the sentence structure. e.g
"..in Javascript apps to manage the data of an application, and was originally designed for React..."
and
"Redux helps you write applications that behave consistently by facilitating and sharing data across the components of an application..."